pipeline/alignments/align.py

```diff
@@ -110,38 +113,60 @@ def decompress(file_path: str):
 def align(
     corpus_src: str,
     corpus_trg: str,
-    priors_input_path: Optional[str],
     tmp_dir: str,
+    chunk_lines: int,
+    priors_input_path: Optional[str],
 ):
     import eflomal
 
-    with ExitStack() as stack:
-        if priors_input_path:
-            logger.info(f"Using provided priors: {priors_input_path}")
-            priors_input = stack.enter_context(open(priors_input_path, "r", encoding="utf-8"))
-        else:
-            priors_input = None
-
-        # We use eflomal aligner.
-        # It is less memory intensive than fast_align.
-        # fast_align failed with OOM in a large white-space tokenized corpus
-        aligner = eflomal.Aligner()
-
-        src_input = stack.enter_context(open(corpus_src, "r", encoding="utf-8"))
-        trg_input = stack.enter_context(open(corpus_trg, "r", encoding="utf-8"))
-        fwd_path = os.path.join(tmp_dir, "aln.fwd")
-        rev_path = os.path.join(tmp_dir, "aln.rev")
-        logger.info("Calculating alignments...")
-        aligner.align(
-            src_input,
-            trg_input,
-            links_filename_fwd=fwd_path,
-            links_filename_rev=rev_path,
-            priors_input=priors_input,
-            quiet=False,
-            use_gdb=False,
-        )
-        return fwd_path, rev_path
+    logger.info("Splitting corpus into parts")
+    # align in chunks to prevent OOM
```

Comment:
Suggestion: Could you document examples of the files this produces? This would help in reasoning with this code.

e.g.

`// Produces chunks of files, like "corpus.en.000", "corpus.en.001"`

or whatever the naming actually is here.